File: binaryTree.js
class BinaryTree {
    constructor(value, left, right) {
        this.value = value;
        this.left = left;
        this.right = right;
    }
}

let mat = [];
function drawTree(tree) {
    if(!tree) return;

    return tree.value + (tree.left) ? drawTree(tree.left): '' + (tree.right) ? drawTree(tree.right): '';
}

tree = new BinaryTree('a', new BinaryTree('b', new BinaryTree('d'), new BinaryTree('e')), new BinaryTree('c', new BinaryTree('f')));
console.log(drawTree(tree));

File: binaryTree.py
class Tree:
    def __init__(self, value, left, right):
        self.value = value
        self.left = left
        self.right = right

mat=[]
def drawTree(tree):
    if tree == None:
        return
    mat.add(tree.value)
    return mat.add([drawTree(tree.left), drawTree(tree.right)])

asci=Tree('a', Tree('b', Tree('d'), Tree('e')), Tree('c', Tree('f')))
print(asci)